model/build-processor/src/main/java/org/keycloak/models/map/processor/GenerateEntityImplementationsProcessor.java

```diff
@@ -363,7 +362,7 @@ public void generate(TypeElement e) throws IOException {
 
                         if (parentMethod.isPresent()) {
                             processingEnv.getMessager().printMessage(Kind.OTHER, "Method " + method + " is declared in a parent class.", method);
-                        } else if (fat != FieldAccessorType.UNKNOWN && ! printMethodBody(pw, fat, method, "f" + me.getKey(), fieldType)) {
+                        } else if (fat == FieldAccessorType.UNKNOWN || ! printMethodBody(pw, fat, method, "f" + me.getKey(), fieldType)) {
```

Comment:
This changes the semantics significantly. We should ignore unknown methods completely as long as the method is not recognized as one of the field accessors.

Reply:
Could you please elaborate more on how this changes the semantics? As far as I can see it just prints a warning message when there is a method with a signature not known to the generator. I added this because during development it is good to know what methods were not recognized.